examples/file-uploader/js/list.js

```diff
@@ -0,0 +1,87 @@
+const Type = require('union-type');
+const T = require('ramda/src/T')
+    , adjust = require('ramda/src/adjust')
+    , append = require('ramda/src/append')
+    , curry  = require('ramda/src/curry')
+;
+const h = require('snabbdom/h');
+
+const upload = require('./upload');
+const uploader = require('./uploader');
+
+const sync = (s) => [s, []];
```

Comment:
`noFx` is good as well. PureScript use the abbreviation "eff".

I also find it a bit cumbersome how the type changes all the way up. But as far as I'm aware that is pretty much how it is in functional programming. If you do a side-effect somewhere down the code then the type change will propagate all the way up.

The Timesheet code is quite horrible. I don't think `update` and `updateE` separation makes things better.